SETUP_ARCH.md

```diff
@@ -0,0 +1,23 @@
+# Snowdrift setup instructions for Arch Linux
+
+Arch has the benefit of being very well-documented, so this document is
+very short!
```

Comment:
Meh, I think it lightens the tone a bit. I can remove it if it's an issue, though.